src/codewhisperer/service/keyStrokeHandler.ts

```diff
@@ -154,6 +154,9 @@ export class KeyStrokeHandler {
         if (!changedRange.isSingleLine || changedText === '') {
             return ''
         }
+        if (changedText.split('\n').length > 1) {
```

Comment:
Does this block Enter auto trigger?

Reply:
oh thanks Lei for the callout, it did block enter auto-trigger

Reply:
will open another PR fixing it